### ISSUE TITLE
Please add the ability to adjust the direction in vertical slide mode

### ISSUE THREAD

**Issue by ghost** [kenwheeler/slick#818]
Thanks

**Comment by nicdford** [comment]
yes please! My use case: We want the direction of our page to feel like you should scroll down, visually we make these distinctions and also having the slider movement following this same momentum would be awesome. Maybe rtl: true could change the direction? That seems somewhat intuitive, except for the naming, since its the equivalent to what you're doing if the slider was horizontal scrolling.

**Comment by elubin** [comment]
:+1:

**Comment by Seanom** [comment]
:+1:

**Comment by godot882** [comment]
+1
Use case: I have a vertical slider with floor plans on a website for a business center. I'd like the ability to reverse the animation so that the user have the feeling he/she going upwards the floors.

I've managed to reverse the indicators with css, but really would like to see a feature for this in the future.

Thanks in advance.

**Comment by stianlp** [comment]
+1

**Comment by jcjmcclean** [comment]
+1

**Comment by Sanster** [comment]
+1

**Comment by hazeways** [comment]
Where do this +1 go?

**Comment by hazeways** [comment]
I want the direction to go down instead of up. you all write +1 but what does it mean?

**Comment by kenwheeler** [comment]
@hazeways it means they are in agreement. An upvote, if you will.

**Comment by hazeways** [comment]
Ah, ok. Thanks Ken for your answer.

**Comment by dev-sumesh** [comment]
I want to move my slick slide in upward and downward directions. Can you please help?

**Comment by vml-cbarendsen** [comment]
+1

**Comment by nomasgrim** [comment]
+1 allow optional direction

**Comment by fr-olivier** [comment]
+1

**Comment by johanguse** [comment]
+1

**Comment by KashanSaleem** [comment]
+1

**Comment by friendrix** [comment]
+1

**Comment by nindik** [comment]
+1

**Comment by TheHanif** [comment]
+100

**Comment by Varin6** [comment]
Is this ever going to happen? I also would like this functionality. Is it not that simple to implement, or what is the reason why it has not yet been done?

**Comment by Varin6** [comment]
Added a pull request here that should solve this issue, please test it.
https://github.com/kenwheeler/slick/pull/2719

**Comment by RussQuint** [comment]
+1, this would be a really good feature for scrolling news feeds - the sense that you're progressing DOWN a list rather than UP one. PS I think this product is great, thank you!

**Comment by Efflamm** [comment]
Did someone tried this ?

`       slidesToScroll: -1,
`

[Slick carousel right to left](https://stackoverflow.com/questions/29770969/slick-carousel-right-to-left)

**Comment by Varin6** [comment]
I have added the reverse vertical slide direction feature and created a pull request (check my previous post above for a link)

Demo is here:
JSFiddle: https://jsfiddle.net/Varin/bk5duvqL/

**Comment by chrislabeard** [comment]
@Varin6 this mostly works, the center class seems to be off a little now though. When in center mode.

**Comment by joeworkman** [comment]
+1

**Comment by joshreisner** [comment]
+1

**Comment by Honza-B** [comment]
+1

**Comment by ADavidLiu** [comment]
+1

**Comment by sangtlee** [comment]
+1

**Comment by DuudeXX8** [comment]
+1

**Comment by k4media** [comment]
> Did someone tried this ?
> 
> ` slidesToScroll: -1,`
> 
> [Slick carousel right to left](https://stackoverflow.com/questions/29770969/slick-carousel-right-to-left)

-1 works for me. Try it!

**Comment by JohnMDDD** [comment]
+1

**Comment by pieterhertogs** [comment]
+1

**Comment by loq24** [comment]
+1

**Comment by TranDuyKhai** [comment]
Has this problem been resolved?

**Comment by alexlinek** [comment]
+1 to add option to switch vertical slide direction. 

It looks like `slidesToScroll: -1` does not work in the current version (1.8.1), and neither does adding `dir="rtl"` to the element with `rtl: true` set. 

@Varin6 's slick update w/ `verticalReverse` works perfectly for me though. Cheers!

**Comment by adnanniazi** [comment]
There is a very simple css solution for this problem. Just rotate the slider direction with CSS like this and it should work like a charm. 

.slick-slider{
transform: rotate(180deg);
}
.slick-slide {
    transform: rotate(-180deg);
}

**Comment by joeworkman** [comment]
I used vertical and then simply reversed my next/prev buttons! 

```
    prevArrow      : ".mb-next",
    nextArrow      : ".mb-prev",
```

The following lines in the verticalReverse PR broke the heights of my slider so I commented those lines out but kept the swipe features. 

```
            _.$slideTrack.css({"display": "flex", "flex-direction": "column-reverse"});
            targetLeft = ((_.slideCount * verticalHeight) * -1) + (slideIndex * - verticalOffset) / _.options.slidesToShow;
```

**Comment by maria-amaltea** [comment]
> There is a very simple css solution for this problem. Just rotate the slider direction with CSS like this and it should work like a charm.
> 
> .slick-slider{ transform: rotate(180deg); } .slick-slide { transform: rotate(-180deg); }

You're a hero!
